core/src/main/java/org/elasticsearch/script/ScriptService.java

```diff
@@ -603,7 +611,8 @@ public void onFileChanged(Path file) {
     public enum ScriptType {
 
         INLINE(0, "inline", "inline", ScriptMode.SANDBOX),
-        INDEXED(1, "id", "indexed", ScriptMode.SANDBOX),
+        // TODO: How to rename with maintaining bwc for ScriptType#scriptType ?
```

Comment:
Given @clintongormley 's comment about it being ok to have a hard break for this, maybe we should remove this TODO now?